web/src/main/java/it/pagopa/selfcare/external_api/web/controller/UserV2Controller.java

```diff
@@ -45,4 +46,21 @@ public UserInfoResource getUserInfo(@ApiParam("${swagger.external_api.user.model
         log.trace("getUserInfo end");
         return userInfoResource;
     }
+
+    @GetMapping(value = "/{id}/onboarded-product")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "", notes = "${swagger.external_api.user.api.getUserProductInfo}")
```

Comment:
you could add `nickname` for setting an appropriate operationId...for ex: `getUserOnboardedProductUsingGET`